setup.py

```diff
@@ -8,12 +19,20 @@
 
 setup(
     name='pyrostest',
-    version='0.0.3',
+    version='0.0.4',
```

Comment:
depends, you should increment on every release, and should increment numbers depending on release type, so A.B.C, increment C for bugfixes, B for major changes, and A for backwards compat breaking changes is iirc how its supposed to be done.